migrations/Version20240327090454.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20240327090454  extends MSPMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add first default policy type and filter type data. Nullable policy value.';
+    }
+
+    protected function getDatabaseType(): ?MSPDatabaseType
+    {
+        return new MSPDatabaseType(MSPDatabaseType::DATABASE_TYPE_GAME_SESSION);
+
+    }
+
+    protected function onUp(Schema $schema): void
+    {
+        // using replace instead of insert to prevent any conflicting data from previous migration versions, of which the inserts have moved here
+        // phpcs:ignoreFile Generic.Files.LineLength.TooLong
+        $this->addSql(
+            <<< 'SQL'
+    ALTER TABLE `policy`
+    CHANGE `value` `value` longtext COLLATE 'utf8mb4_bin' NULL DEFAULT 'NULL' COMMENT '(DC2Type:json)' AFTER `type_id`;    
```

Comment:
The buffer zone policy need a range value, but not all policies need a value. So the value of a policy can be null. eg. seasonal bans .